storage/storage_tests/storage_tests.cpp

```diff
@@ -792,14 +794,45 @@ UNIT_TEST(StorageTest_CancelDownloadingWhenAlmostDone)
   TEST(!file, (*file));
 }
 
-UNIT_TEST(StorageTest_DeleteCountry)
+UNIT_TEST(StorageTest_DeleteCountrySingleMwm)
+{
+  Storage storage(COUNTRIES_MIGRATE_FILE, kMapTestDir);
```

Comment:
A lot of tests here have following common code at the beginning:
```c++
Storage storage;
TaskRunner runner;
InitStorage(storage, runner);
...
```
You can easily get rid of this code by using UNIT_CLASS_TEST instead of UNIT_TEST. Just create following:
```c++
class StorageMigrateTest
{
public:
  StorageMigrateTest(): m_storage(COUNTRIES_MIGRATE_FILE, kMapTestDir)
  {
     // storage initialization code.
  }

protected:
  Storage m_storage;
  TaskRunner m_taskRunner;
};
...

UNIT_CLASS_TEST(StorageMigrateTest, DeleteCountrySingleMwm)
{
   // use m_storage and m_runner here.
}
```